sources/Interop/Windows/um/wslapi/Windows.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/wslapi.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public static unsafe partial class Windows
+    {
+        [DllImport("api-ms-win-wsl-api-l1-1-0", EntryPoint = "WslIsDistributionRegistered", ExactSpelling = true)]
```

Comment:
This might be a docs issue, I think the actual assembly is meant to be `wslapi`

Reply:
I believe that they are just recommending that you go through the API Set, but I can change it to `wslapi`